pypsa/plot.py

```diff
@@ -255,14 +265,26 @@ def as_branch_series(ser):
 
     if basemap_present:
         ax.axis('off')
-    if cartopy_present and drawmap:
+    if cartopy_present and geomap:
         ax.outline_patch.set_visible(False)
 
     ax.set_title(title)
 
     return (bus_collection,) + tuple(branch_collections)
 
 
+def get_projection_from_crs(crs):
+    if crs == 4326:
+        # if data is in latlon system, return default map with latlon system
+        return ccrs.PlateCarree()
+    try:
+        return ccrs.epsg(crs)
+    except ValueError:
+        logger.warning(f"'{crs}' does not define a projected coordinate system. "
+                    "Falling back to latlong.")
+        return ccrs.PlateCarree()
+
+
 def compute_bbox_with_margins(margin, x, y):
```

Comment:
This function should be used to assoiciate the network.srid with an proper projection. As the srid=4326, which is not an valid projection for cartopy, refers to the lat/lon system, the function takes the default lat/lon projection PlateCarree in this case, without a waring, as otherwise, the warning would appear many times for the default srid.